app/locale.js

```diff
@@ -312,62 +312,87 @@ exports.translation = function (token, replacements = {}) {
 }
 
 // Default language locale identifier
-const DEFAULT_LANGUAGE = 'en-US'
+const DEFAULT_LANGUAGE = lang
 
-const availableLanguages = [
-  'bn-BD',
-  'bn-IN',
-  'cs',
-  'de-DE',
-  'en-GB',
-  'en-US',
-  'es',
-  'eu',
-  'fr-FR',
-  'hi-IN',
-  'id-ID',
-  'it-IT',
-  'ja-JP',
-  'ko-KR',
-  'ms-MY',
-  'nl-NL',
-  'pl-PL',
-  'pt-BR',
-  'ru',
-  'sl',
-  'sv-SE',
-  'ta',
-  'te',
-  'tr-TR',
-  'uk',
-  'zh-CN'
-]
+// locale brave
+var availableLanguages = []
 
-// Currently configured languages
-const configuredLanguages = {}
-availableLanguages.forEach(function (lang) {
-  configuredLanguages[lang] = true
-})
+const configuredLanguages = {
+  // locale electron : locale brave
+  'eu':'eu',
+  'bn':'bn-BD',
+  'bn-BD':'bn-BD',
+  'bn':'bn-IN',
+  'bn-IN':'bn-IN',
+  'zh-CN':'zh-CN',
+  'cs':'cs',
+  'nl':'nl-NL',
+  'nl':'nl',
+  'en-US':'en-US',
+  'en':'en-GB',
+  'en-GB':'en-GB',
+  'en-AU':'en-GB',
+  'en-CA':'en-GB',
+  'en-NZ':'en-GB',
+  'en-ZA':'en-GB',
+  'fr-FR':'fr-FR',
+  'fr':'fr-FR',
+  'fr-FR':'fr-FR',
+  'fr-CA':'fr-FR',
+  'fr-CH':'fr-FR',
+  'de-DE':'de-DE',
+  'de':'de-DE',
+  'de-AT':'de-DE',
+  'de-CH':'de-DE',
+  'hi':'hi-IN',
+  'hi-IN':'hi-IN',
+  'id':'id-ID',
+  'id-ID':'id-ID',
+  'it':'it-IT',
+  'it-IT':'it-IT',
+  'it-CH':'it-IT',
+  'ja':'ja-JP',
+  'ja-JP':'ja-JP',
+  'ko':'ko-KR',
+  'ko-KR':'ko-KR',
+  'ms':'ms-MY',
+  'ms-MY':'ms-MY',
+  'pl':'pl-PL',
+  'pl-PL':'pl-PL',
+  'pt':'pt-BR',
+  'pt-BR':'pt-BR',
+  'pt-PT':'pt-BR',
+  'ru':'ru',
+  'sl':'sl',
+  'sv':'sv-SE',
+  'sv-SE':'sv-SE',
+  'es':'es',
+  'es-419':'es',
+  'ta':'ta',
+  'te':'te',
+  'tr':'tr-TR',
+  'tr-TR':'tr-TR',
+  'uk':'uk'
+}
+
+// Dynamic filling of the  array of locale
+for (var local in configuredLanguages) {
+	if (availableLanguages.indexOf(configuredLanguages[local]) === -1) {
+		availableLanguages.push(configuredLanguages[local])
+	}
+}
```

Comment:
i think since we support es6 you can just replace this block with `availableLanguages = [...new Set(Object.values(configuredLanguages))]`

Reply:
effectively,
Thank you